scripts/generate-icon-font.ts

```diff
@@ -0,0 +1,63 @@
+import webfont from 'webfont';
+import fs from 'fs/promises';
+import { GlyphData } from 'webfont/dist/src/types';
+
+/** Icons to include in the generated icon font */
+const INCLUDED_ICONS = ['connection-active', 'connection-inactive'];
+
+/**
+ * Generates an icon font from the included icons and outputs package.json
+ * configuration field for those icons as specified in
+ * https://code.visualstudio.com/api/references/icons-in-labels
+ */
+async function main(): Promise<void> {
+  const font = await webfont({
```

Comment:
I think it's worth it to have a external dev dependency / script for this. We can potentially utilize Vscode's support for icons fonts for i.e. easily having different color versions of the same icons and, as in this case, easily including icons in different places.
The link from the comment:
https://code.visualstudio.com/api/references/icons-in-labels